src/app/plugins/pubsub.plugin.ts

```diff
@@ -125,13 +132,34 @@ export class PubSubPlugin extends Plugin {
     return embed;
   }
 
+  private _parseSub(subData: any): subData {
+    const name = this._normalizeName(subData.sub_name) || 'Nothing Found';
+    const onSale = subData.status.toLowerCase() === 'true';
+    const dates = subData.last_sale || ' - ';
+    const image = subData.image || '';
+    const price = subData.price || '';
+
+    const [startDate, endDate, _] = dates.split('-');
+    console.log('test');
+
+    return {
+      image: image,
+      name: name,
+      price: price,
+      onSale: onSale,
+      saleDates: dates,
+      startDate: startDate,
+      endSaleDate: endDate,
+    };
+  }
+
```

Comment:
This should take in a `subData` and return a `subData` just modify the values inside of it

Reply:
but I've constantly been told not to edit the args 🙃

Reply:
Thats not an arg